tf2onnx/onnx_opset/nn.py

```diff
@@ -481,6 +481,7 @@ def version_7(cls, ctx, node, **kwargs):
         # wants the input to be NHWC - adjust target_shape to this.
         n, h, w, c = shape
         nh, nw = target_shape
+        utils.make_sure(all(i != -1 for i in [nh, nw]), "h and w need to be known")
```

Comment:
please use utils.ONNX_UNKNOWN_DIMENSION

Reply:
converter has the code "utils.ONNX_UNKNOWN_DIMENSION = args.unknown_dim"
if user specifies unknown_dim as 1, then ....